gradle/coverage/python-coverage.gradle

```diff
@@ -0,0 +1,18 @@
+//coverage related args to be passed to pytest
+ext.get_coverage_args = { test_name = "" ->
+
+  def coverage_file_name = "pycov-${project.name}${test_name}.xml"
+
+  /*
+  Tools that aggregate and analyse coverage tools search for the coverage result files. Keeping them under one folder
+  will minimize the time spent searching through the full source tree.
+  */
+  def base_path = "${rootProject.buildDir}/coverage-reports"
+
+  /*
+  --cov=src was added via setup.cfg in many of the python projects but for some reason, was not getting picked up
+  consistently, so adding it explicitly. Centralizing these params would make it easier to adjust them for all python
+  projects (with overrides being in the sub-project build script.)
```

Comment:
The `--cov=src` thing being required is a bit worrying. In general, I prefer having stuff in configs instead of specifying things on the command line. We can update the setup.cfg files in the python directories if that's necessary

That said, if you spent a bunch of time messing with this and this was the only way you got stuff to work, then it's not a blocker

Reply:
I'll relook at this another time. I think the cause is related to the equivalent of https://github.com/chakru-r/datahub/blob/master/metadata-ingestion/setup.cfg#L107-L116 across the other python projects. 

That said, it may help if some of these settings - especially those that are common should not be required to be specified in each project - only overrides or deviations are added if applicable.